### PR TITLE
Remove progressQ

### PR DESCRIPTION
**Pull request by poncovka**
Remove the support for the progress queue.
Use progress reporting of the new task instead.

### CODE REVIEW

File: pyanaconda/ui/tui/spokes/installation_progress.py [hunk 1]
```diff
@@ -61,74 +59,58 @@ def completed(self):
...
+        task.progress_changed_signal.connect(
+            self._on_progress_changed
+        )
         task.start()
 
-        # This will run until we're all done with the install thread.
-        self._update_progress()
+        # This will run until the task is finished.
+        while task.is_running:
+            sleep(1)
```
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/installation_progress.py:100]: @jkonecny12, is it possible implement a solution without the sleep? Does `simpleline` support something like that?

File: pyanaconda/ui/tui/spokes/installation_progress.py [hunk 2]
```diff
@@ -61,74 +59,58 @@ def completed(self):
...
+
+            # Also flush the communication Queue at least once a second and
+            # process it's events so we can react to async evens (like a thread
+            # throwing an exception)
+            loop = App.get_event_loop()
+            loop.process_signals()
```
Comment (VladimirSlavik) [inline pyanaconda/ui/tui/spokes/installation_progress.py:106]: Is this a safe thing to do?

Comment (poncovka) [reply pyanaconda/ui/tui/spokes/installation_progress.py:106]: It is nothing new. You can find the same lines in the previous solution.

File: pyanaconda/ui/tui/spokes/installation_progress.py (hunk 1 above)
Comment (jkonecny12) [reply pyanaconda/ui/tui/spokes/installation_progress.py:100]: You need to wait on something so if you have a source you can use signals if I'm not mistaken. Do we have something like that?

Comment (jkonecny12) [reply pyanaconda/ui/tui/spokes/installation_progress.py:100]: There is `return_after` parameter for this method which will end when a specific signal arrive so if you have that it should be pretty straightforward.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

File: pyanaconda/ui/tui/spokes/installation_progress.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/ui/tui/spokes/installation_progress.py]: Thanks! I have used it.

**Comment by poncovka** [comment]
I have updated the "[Don't use progressQ in TUI](https://github.com/rhinstaller/anaconda/pull/3886/commits/050a2865c77424861240dbcd6a38ed7d04cdee29)" commit.

**Review by jkonecny12 — APPROVED** [review]
I like these improvements. Thanks!

**Pull request merged**